docs/updating/from_1.x_2.x/update_app_to_2.5.md

```diff
@@ -13,16 +13,16 @@ latest_tag: '2.5.27'
 
 [[% include 'snippets/update/merge_composer.md' %]]
 
-## 3. Update the app
+## 3. Version Specific changes
 
-First, perform version-specific steps depending on which version you are updating from.
+When updating a release **earlier than v2.5.0**, some version-specific changes are required.
 
-### A. v2.2
+### A. Update from <=v2.1.1
```

Comment:
I would prefer
```suggestion
### A. Update from earlier than v2.2
```
If not, maybe the use of  `≤` instead of `<=`
```suggestion
### A. Update from ≤v2.1.1
```

Reply:
- I used "<="  because I have seen it here https://doc.ibexa.co/en/2.5/updating/from_1.x_2.x/update_app_to_2.5/#c-prepare-for-pulling-changes
- "earlier than" is less specific than "<=v2.1.1"

but I am definitely fine with "from earlier than v2.2" (or v2.2.0)